README.md

```diff
@@ -156,8 +171,8 @@ We suggest the following file layout for JSON schema related files:
 /docs (top-level directory for project documentation)
   /schema (API schema documentation)
     /schemata
-      /{resource.json} (individual resource schema)
-    /meta.json (overall API metadata)
+      /{resource.{json,yml}} (individual resource schema)
```

Comment:
Seems fine, though I think I might use square brackets instead of curly here, so more like `{resource.[json,yml]}`. Not sure though, what do you think?